docs/css/styles.css

```diff
@@ -1,2 +1,9 @@
 .details .open {
 }
+
+.tabbed-content {
```

Comment:
```suggestion
.tabbed-set {
```

Accept this suggestion if you prefer this:

![image](https://github.com/user-attachments/assets/2601084f-60c9-4c23-84f1-74816319c57c)

I think I do :)